gemrb/GUIScripts/GUISTORE.py

```diff
@@ -188,7 +187,6 @@ def ChangeStoreView(func):
 	Store = GemRB.GetStore ()
 	#based on shop type, these buttons will change
 	store_buttons = Store['StoreButtons']
-	BarteringPC = GemRB.GameGetFirstSelectedPC ()
```

Comment:
Did this update the actor before, if you changed the selection while in store? Now we explicitly only work with the initiator.